doc/contributing.md

```diff
@@ -326,18 +326,19 @@ begin with `>>>`.  To keep this example code from being run during testing, add
 
 Below the import statements at the top of the file
 
-``
+```
 __doctest_skip__ = ["module_name"]
```

Comment:
Here we write `modue_name`, but in the text above we have `function name` (please note the missing underscore) and later in the sentence `module`:
https://github.com/GenericMappingTools/pygmt/blob/5e4388bb37248ede4b0fae2d241d34bf4636155b/doc/contributing.md?plain=1#L318-L324

 I fell we should be a bit more consistent.

```suggestion
```python
__doctest_skip__ = ["function_name"]
```

We also have methods with inline examples, but maybe using "function" is OK here?
Or should we re-writte this part to something like (not 100 % sure if `function_method_name` is a good name):
```markdown
Many of the PyGMT functions and methods have example code in their documentation. To
contribute an example, add an "Example" header and put the example code below it. Have
all lines begin with `>>>`. To keep this example code from being run during testing, add
the code `__doctest_skip__ = [function_method_name]` to the top of the function or method.
```

```suggestion
```python
__doctest_skip__ = ["function_method_name"]
```

Reply:
Fixed in af2f6d62b3e3e2cb2ac95b51147dd20e337cd9a2.